src/aws.js

```diff
@@ -42,6 +30,34 @@ async function startEc2Instance(label, githubRegistrationToken) {
   }
 }
 
+// User data scripts are run as the root user
+function buildUserDataScript(githubRegistrationToken, label) {
+  if (config.runnerHomeDir) {
```

Comment:
thanks for this PR! I've been trying it out, and I think this needs to be `config.input.runnerHomeDir`

Reply:
Fixed